### ISSUE TITLE
Error in the reference - see Remark for `metadata -> revision`

### ISSUE THREAD

**Issue by iMichaela** [usnistgov/OSCAL#1927]
The Remark statement of `metadata -> revision` wrongly indicates that `last-modified` and `oscal-version` are not required elements. See blue highlighted text in the image below. Only `published` is an optional element.

I suggest the following corrected text (but any other accurate content will work):

**While `published` is not required, a value for this entry should be provided if the information is known, in addition to the mandatory elements `last-modified`, and `oscal-version`.**

<img width="912" alt="Screenshot 2023-09-05 at 3 10 34 PM" src="https://github.com/usnistgov/OSCAL-Reference/assets/13984545/2d05571d-feb6-4c67-95da-8208c9f89762">

**Comment by aj-stein-nist** [comment]
@Arminta-Jenkins-NIST, I assigned this one to you since you volunteered to pick this up. Thanks again.

**Comment by iMichaela** [comment]
@Arminta-Jenkins-NIST - feel free to consider the suggested fix above, but any other accurate content will work:
Text could read:
"While published is not required, a value for this entry should be provided if the information is known, in addition to the mandatory elements last-modified, and oscal-version."

**Comment by aj-stein-nist** [comment]
Per discussion with Arminta and the team during standup, Arminta's analysis is correct. We discussed the potential of adding verbiage to the remarks to explain the relevant fields are reused but with different cardinality in `/*/metadata/revisions` as opposed to `/*/metadata`, but I warned that is done so frequently in OSCAL models by leveraging the modular reuse in Metaschema, we are opening ourselves up to sprinkling that commentary into many, many places across models. The added benefit is unclear, so we agreed during standup it is ok to close this one.

Some good takeaways about this can be had during retro about how I brought it into the sprint quickly without doing these checks myself, so kudos to the team for catching me in the act. 😅
